classes/helper/HelperList.php

```diff
@@ -124,11 +128,23 @@ class HelperListCore extends Helper
 
     public $page;
 
+    /** @var EntityLinkBuilderFactory */
+    private $linkBuilderFactory;
+
     public function __construct()
     {
         $this->base_folder = 'helpers/list/';
         $this->base_tpl = 'list.tpl';
 
+        $adminLinkBuilder = new AdminLinkBuilder(Context::getContext()->link, [
```

Comment:
@eternoendless I adressed your feedback
I was wondering about this initialization, maybe it would be handy to have this info about migrated controllers somewhere in a configuration? To centralize this information which might be used somewhere else, or in a future service definition

Reply:
It could be a class constant.

Reply:
Yes, but I wonder which would be the best candidate to store this const? Should we use `AdminLinkBuilder` for now? or do you think of a more generic/transversal class?

Reply:
I think we can leave it as is for now.